climpred/bootstrap.py

```diff
@@ -51,6 +51,44 @@ def _resample(hind, resample_dim):
     return smp_hind
 
 
+def _resample_idx(init, bootstrap, dim='member', replace=True):
+    """Resample bootstrap times over dim."""
+
+    def xr_broadcast(x, idx):
```

Comment:
I know we've been using this name, but I'd rename this to be more clear. Broadcast is a pretty general term. We're just using this to select different indices from a single dimension many times. Maybe `select_bootstrap_indices_ufunc` or something similar.

Also add a brief docstring here since the line of code is not really obvious what it's doing.

"""Selects indices of bootstrapped dimension over all iterations."""